packages/terra-date-picker/src/orion-fusion-theme/react_datepicker.module.scss

```diff
@@ -7,11 +7,11 @@
     --terra-date-picker-day-focus-outline: 2px dashed #000;
     --terra-date-picker-day-focus-outline-offset: 0;
 
-    @include terra-inline-svg-var('--terra-date-picker-prev-month-background-image', '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="rgb(204, 204, 204)" d="M36 48L12 24 36 0v48z"></path></svg>');
+    @include terra-inline-svg-var('--terra-date-picker-prev-month-background-image', '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="rgba(28, 31, 33, 0.85)" d="M36 48L12 24 36 0v48z"></path></svg>');
     @include terra-inline-svg-var('--terra-date-picker-prev-month-focus-background-image', '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="rgb(179, 179, 179)" d="M36 48L12 24 36 0v48z"></path></svg>');
```

Comment:
Does the color for the focus style also need to be updated or is it ok to leave it light gray like how it was before?

![Screen Shot 2020-07-21 at 2 27 04 PM](https://user-images.githubusercontent.com/22618655/88097760-47315080-cb5e-11ea-9274-002b1bf4cbba.png)

Reply:
updated here https://github.com/cerner/terra-framework/pull/1182/commits/d1a05b9b062ee3e28d88b7fa0c308d891e660904.

![Screenshot 2020-07-22 at 12 45 57 PM](https://user-images.githubusercontent.com/29273628/88146732-df404000-cc19-11ea-90d0-bda7d955854b.png)